bazel/repositories_extra.bzl

```diff
@@ -14,6 +14,10 @@ def _python_deps():
         name = "protodoc_pip3",
         requirements = "@envoy//tools/protodoc:requirements.txt",
     )
+    pip3_import(
```

Comment:
Just to confirm the pip3 trick is so you can bring in Clang as a Python dependency during test via requirements.txt? @lizan @PiotrSikora is there going to be some Wasm dependency that eventually gives us this more directly? I think what is here right now is fine for now though.

Reply:
yes I have to install the `clang` module so that I can use them at test.